apps/masters/views.py

```diff
@@ -525,3 +526,174 @@ def create(self, request, *args, **kwargs):
 
     def update(self, request, *args, **kwargs):
         return update_instance(self, request, *args, **kwargs)
+
+#===============================================PDF_creation================================
+from reportlab.platypus import SimpleDocTemplate, Paragraph, Spacer
+from reportlab.lib.styles import getSampleStyleSheet,ParagraphStyle
+from apps.masters.utils.table_defination import create_table_1, create_table_2, create_table_3, create_table_4, create_table_5, create_table_6
+from config.utils_methods import convert_amount_to_words, extract_product_data, format_phone_number,send_pdf_via_email, get_related_data, list_all_objects, create_instance, update_instance, build_response
+from django.http import Http404
+from apps.customer.models import CustomerAddresses
+from .utils.docs_variables import doc_data
+from reportlab.lib.pagesizes import inch
+
```

Comment:
Move these imports top of the file, these shouldn't be in the middle

Reply:
Moved